docs/query_cost.md

```diff
@@ -0,0 +1,118 @@
+guaranteed that per-chromosome queries will only process variants of the
+chromosome under study. 
+
+As a concrete example, during the process of publishing
+[gnomAD on GCP marketplace](https://console.cloud.google.com/marketplace/details/broad-institute/gnomad)
+we imported [gnomAD v3](https://gnomad.broadinstitute.org/) VCF files into 
+a single BigQuery table with 707,950,943 rows which occupied 1.1 TB storage. 
+After splitting by chromosome, we ended up with smaller tables for each chromosome,
+here are some of their sizes:
+
+| Chromosome | Number of Rows  |  Size (GB) |
+|------------|-----------------|------------|
+| Chr1       | 55,215,812      | 88.31      |
+| Chr2       | 58,605,556      | 94.01      |
+| Chr3       | 47,688,920      | 77.78      |
+| Chr21      | 10,248,793      | 15.62      |
+| Chr22      | 10,882,958      | 17.78      |
+| ChrY       | 1,344,278       | 1.64       |
+| ChrX       | 28,156,663      | 40.51      |
+
+
+Other than making each table smaller and more manageable, sharding per chromosome
+is a prerequisite for the next cost reduction technique.
+
+## (II) BigQuery Partitioning
+ 
+BigQuery [partitioning](https://cloud.google.com/bigquery/docs/partitioned-tables#integer_range)
+is a technique that divides a large table into segments, called partitions,
+this makes it easier to manage and query your data. By dividing a large table into
+smaller partitions, you can improve query performance, and you can control costs by
+reducing the number of bytes read by a query.
+
+As we mentioned earlier, since BigQuery uses a columnar data structure, price of a
+query is set according to the total data processed in the columns you select.
+For large tables, such as gnomAD, this cost will be very significant, especially
+if we are querying a small region of genome. 
+
+Partitioning is the perfect solution here: BigQuery considers the `WHERE` clause
+and based on that it only processes the set of partitioning that contain rows that
+are relevant to the query. Following figure schematically shows the cost of previous
+query on a table which is partitioned based on of `start_position` column :
+
+![query cost unpartitioned table](images/query_cost_partitioned_table.svg)
+
+Partitioning is a very effective technique to reduce the cost of queries, specially
+for the genomics data where majority of the queries are analyzing a small genomic region.
+Table partitioning comes at no cost, and allows to divide a table up to 4000
+segments. Appending to a paritioned table will not degrade the performance and all newly
+added rows will be stored in the right partition. Also you can know query cost before
+running your query. Because partition pruning is done before the query runs, 
+you can get the query cost through a [dry run](https://cloud.google.com/bigquery/docs/dry-run-queries). 
+
+## (III) BigQuery Clustering
+
+[BigQuery clustering](https://cloud.google.com/bigquery/docs/clustered-tables)
```

Comment:
I would drop all of this and just talk about the clustering we perform.  If they want to know more, they can read the BQ docs.

Reply:
I agree, we better keep this section as small as possible and refer users to BigQuery docs.
If you don't mind I will leave this for Ben to do.